My build fails because of HDWalletKit

After installing HDWalletKit via cocoaPods, my build fails with following error:

```

HDWalletKit/HDWalletKit/Core/BitcoinScript/Script.swift:455:107: error: 
the compiler is unable to type-check this expression in reasonable time; 
try breaking up the expression into distinct sub-expressions

```

In development I have fixed this by breaking up mentioned line of code into 2
But now that I want to perform `pod lint lib` or publish it, it fails
both on my machine and on travis

--- a/HDWalletKit/Core/BitcoinScript/Script.swift
+++ b/HDWalletKit/Core/BitcoinScript/Script.swift
@@ -452,9 +452,12 @@ extension Script {
     // Standard Transaction to Bitcoin address (pay-to-pubkey-hash)
     // scriptPubKey: OP_DUP OP_HASH160 <pubKeyHash> OP_EQUALVERIFY OP_CHECKSIG
     public static func buildPublicKeyHashOut(pubKeyHash: Data) -> Data {
-        let tmp: Data = Data() + OpCode.OP_DUP + OpCode.OP_HASH160 + UInt8(pubKeyHash.count) + pubKeyHash
-        let anotherTmp = tmp +  OpCode.OP_EQUALVERIFY
-        return anotherTmp + OpCode.OP_CHECKSIG
+        let values: [[UInt8]] = [
+            [OpCode.OP_DUP.value, OpCode.OP_HASH160.value, UInt8(pubKeyHash.count)],
+            pubKeyHash.bytes,
+            [OpCode.OP_EQUALVERIFY.value,  OpCode.OP_CHECKSIG.value]
+        ]
+        return Data(values.reduce([], +))
     }
 
     public static func buildPublicKeyUnlockingScript(signature: Data, pubkey: PublicKey, hashType: SighashType) -> Data {

--- a/HDWalletKit/Core/BitcoinScript/Script.swift
+++ b/HDWalletKit/Core/BitcoinScript/Script.swift
@@ -453,8 +453,8 @@ extension Script {
     // scriptPubKey: OP_DUP OP_HASH160 <pubKeyHash> OP_EQUALVERIFY OP_CHECKSIG
     public static func buildPublicKeyHashOut(pubKeyHash: Data) -> Data {
         let tmp: Data = Data() + OpCode.OP_DUP + OpCode.OP_HASH160 + UInt8(pubKeyHash.count) + pubKeyHash
-        return tmp + OpCode.OP_EQUALVERIFY + OpCode.OP_CHECKSIG
-
+        let anotherTmp = tmp +  OpCode.OP_EQUALVERIFY
+        return anotherTmp + OpCode.OP_CHECKSIG
     }
 
     public static func buildPublicKeyUnlockingScript(signature: Data, pubkey: PublicKey, hashType: SighashType) -> Data {

--- a/HDWalletKit_Tests/CryptoTests.swift
+++ b/HDWalletKit_Tests/CryptoTests.swift
@@ -98,4 +98,15 @@ class CryptoTests: XCTestCase {
         let receive = account.derived(at: .notHardened(0))
         return receive
     }
+
+     func testPublickKeyHashOutFromPubKeyHash() {
+        let expected = "76a9210392030131e97b2a396691a7c1d91f6b5541649b75bda14d056797ab3cadcaf2f588ac"
+        let entropy = Data(hex: "000102030405060708090a0b0c0d0e0f")
+        let mnemonic = Mnemonic.create(entropy: entropy)
+        let seed = Mnemonic.createSeed(mnemonic: mnemonic)
+        let privateKey = PrivateKey(seed: seed, coin: .bitcoin)
+        let publicKey = privateKey.publicKey.data
+        let hash = Script.buildPublicKeyHashOut(pubKeyHash: publicKey)
+        XCTAssertEqual(hash.toHexString(), expected)
+    }
 }

